src/home_robot/setup.py

```diff
@@ -18,7 +18,6 @@
     "sophuspy",
     "trimesh",
     "pin==2.6.17",
-    "torch_geometric",
```

Comment:
Not sure if this can be completely removed. The voxel code uses this https://github.com/facebookresearch/home-robot/blob/main/src/home_robot/home_robot/utils/voxel.py and the OVMM evaluations do import this